bin/stringmerge

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+'''
+Merge two existing problems.
+'''
+
+import sys
+import argparse
+
+from stringfuzz.ast import ExpressionNode, SortNode, IdentifierNode, FunctionDeclarationNode, SortedVarNode
+from stringfuzz.ast_walker import ASTWalker
+from stringfuzz.constants import LANGUAGES, SMT_25_STRING
+from stringfuzz.generator import generate
+from stringfuzz.parser import parse, ParsingError
+from stringfuzz.smt import smt_string_logic, smt_sat
+
+# defaults
+DEFAULT_RENAME_IDS = False
+
+# constants
+DECLARE_CONST = "declare-const"
+
+# helper functions 
+def assertion(expr):
+    if isinstance(expr, ExpressionNode):
+        symbol = expr.symbol
+        if isinstance(symbol, IdentifierNode) and symbol.name == "assert":
+            return True
+    return False
+
+def declaration(expr):
+    if isinstance(expr, (FunctionDeclarationNode, SortedVarNode)):
+        return True
+    if isinstance(expr, ExpressionNode):
+        symbol = expr.symbol
+        if isinstance(symbol, IdentifierNode) and symbol.name == DECLARE_CONST:
+            return True
+    return False
+
+def sort(expr):
```

Comment:
Nitpick: for these functions, please rename them to `is_X` instead of just `X` to help avoid name conflicts.